lua/fspectate/cl_init.lua

```diff
@@ -512,6 +514,22 @@ local function drawHelp()
     draw.WordBox( 2, bottomLeft.x, bottomLeft.y + 12, "Left click to spectate!", "UiBold", uiBackground, uiForeground )
 end
 
+--[[---------------------------------------------------------------------------
+Draws a crosshair
+---------------------------------------------------------------------------]]
+local function drawCrosshair()
+    if not showCrosshair then return end
+    local h = ScrH() / 2
+    local w = ScrW() / 2 - 1 -- -1 to make it exactly match with the hl2 crosshair.
```

Comment:
🤨

Reply:
What's the issue? try it for yourself, it really does need the -1 due to sorse.